data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -0,0 +1,10 @@
+#cc-oci-runtime
+
+This source rpm is the clear-containers runtime used to create clear-containers. 
+To update and generate an rpm spec and dsc to create rpm and deb packages, 
```

Comment:
I am editing this file but this sentence is undecipherable. Please avoid using so many abbreviations in a sentence. If the script performs multiple operations with outputs, consider using a list. I believe this to be the case.